### PR TITLE
update to custom_lfn function

### PR DESCRIPTION
**Pull request by datikavt**
Create custom_lfn function to get LFNs of custom Datasets

### CODE REVIEW

File: columnflow/tasks/external.py [hunk 1]
```diff
@@ -103,7 +103,8 @@ def run(self):
         lfns = []
         for key in sorted(self.dataset_info_inst.keys):
             self.logger.info(f"get lfns for dataset key {key} {msg}")
-            lfns.extend(get_dataset_lfns(self.dataset_inst, self.global_shift_inst, key))
+            # lfns.extend(get_dataset_lfns(self.dataset_inst, self.global_shift_inst, key))
```
Comment (JanvanderLinden) [inline columnflow/tasks/external.py]: Doesnt this break other calls of this function? It seems like the default arguments used for other `get_dataset_lfns` calls also has the `self.global_shift_inst` argument.

Comment (datikavt) [reply columnflow/tasks/external.py]: Thanks, should be good now

Comment (JanvanderLinden) [reply columnflow/tasks/external.py]: dont you have 
```
self.dataset_inst, self.global_shift_inst
```
also for your implementation? Shouldnt that be safer to use the same initialization?

Comment (datikavt) [reply columnflow/tasks/external.py]: The same initialization didn't really work for some reason.

Comment (JanvanderLinden) [reply columnflow/tasks/external.py]: Ok, then lets leave it as it is now

**Pull request merged**